bigframes/session/__init__.py

```diff
@@ -755,15 +757,25 @@ def _read_gbq_table(
             )
 
         # ----------------------------------------------------
-        # Create Block & default index if len(index_cols) == 0
+        # Create Default Sequential Index if still have no index
         # ----------------------------------------------------
 
+        if not index_col and len(index_cols) == 0:
```

Comment:
This reads odd to me: `len(index_cols) == 0` implies `not index_col`. Could you rephrase what you're trying to check here? I assume you're trying to exclude the DefaultIndexKind enum from this check. Let's be explicit about that.

Reply:
yeah, basically, trying to fall back to sequential index if don't have null index, user provided index columns, or metadata-derived index columns. Rewrote the condition though with the code structure, its still a bit weird.